modules/pel/peltool/peltool.py

```diff
@@ -860,6 +860,7 @@ def main():
         config.extension = args.extension
 
     if args.file:
+        config.every_pel = True
```

Comment:
There's a 'ebmcdumputil' tool at https://github.ibm.com/openbmc/openbmc-tools/blob/1020/ebmcdump/ebmcdumputil#L330  that lets the user select if they want all PELs or just serviceable only that would still need to work.  That script can be updated if necessary.